cypress/automated-tests/blind/required-fields.cy.js

```diff
@@ -20,20 +20,10 @@ describe('blind required fields', () => {
       .get('[data-field-code="LanguageSelection"]')
       .contains('English')
       .click();
-    cy
-      .get('#form-section-0 > .form-section-buttons > .form-section-next')
-      .click();
-    
-    // Instructions
-    cy
-      .get('#form-section-1 > .form-section-buttons > .form-section-next')
-      .click();
   });
 
   it("does not fill in the application type and sees an error", function() {
-    cy
-      .get('#form-section-2 > .form-section-buttons > .form-submit-button')
-      .click()
+    cy.get('.form-submit-button').click();
 
     cy.get('.required-text').should('be.visible');
```

Comment:
Question: Now that the form is one page, are we at risk of false negatives if we are looking for any `.required-text` elements? Do we need to use more targeted selectors to ensure that the user sees the error under the correct fields, something like the `within()` function used in the youth pass tests?
`        cy.get('#form-element-wrapper_242').within(() => {
            cy.get('.required-text').should('be.visible');
        });`

Reply:
That is a great point. In addressing this, I also realized I could make the structure of the test simpler because I could test everything at once given that it was a single page.